tools/osbuilder/tests/test_config.sh

```diff
@@ -34,30 +34,28 @@ distro_in_set() {
 	return 1
 }
 
-if [ -n "${CI:-}" ]; then
-	# Since too many distros timeout for now, we only test clearlinux and ubuntu. We can enable other distros when we fix timeout problem.
-	for distro in "${distros[@]}"; do
-		if distro_in_set "${distro}" "${test_distros[@]}"; then
-			continue
-		fi
-		skipWhenTestingAll+=("${distro}")
-	done
+# Since too many distros timeout for now, we only test clearlinux and ubuntu. We can enable other distros when we fix timeout problem.
```

Comment:
It seems wrong to keep this code that is supposed to only be run when CI is set. Can you clarify @GabyCT ?

Reply:
@gkurz @sprt thanks for the feedback changes applied, sorry that I missed that